finetune/datasets/summaries_unsupervised_mlm.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from newsroom import jsonl
+import random
+from finetune import Classifier
+import ujson as _json
+import glob
+
+logging.basicConfig(level=logging.DEBUG)
+DATASET_PATH = "/root/code/Data/newsroom/train_half.data"
```

Comment:
If we do ship this in the datasets folder I think we should probably train on something small / naive, maybe with trimmed down model size as a toy example.

Reply:
I would feel weird redistributing a portion of the summaries dataset... need to at least check licensing.